doc/project/building.md

```diff
@@ -11,13 +11,13 @@ This will generate `.sjsir` and `.class` files for each class in your project (j
 
 The `.sjsir` files are an internal representation which can be linked to actual JavaScript code as will be explained shortly.
 
-## Produce one JavaScript file
+## Produce JavaScript
```

Comment:
The changes in this section are not related to dropping the doc of 0.6.x. They should be in a separate PR.

Reply:
Ouch, good catch. This was me trying to update the doc quickly :P